### PR TITLE
Publish compiled Protobuf types to NPM

### PR DESCRIPTION
**Pull request by dmytro-grankin**
This PR brings publishing of Protobuf types compiled to JavaScript.

The files are published to NPM as `spine-users` package. The actual publishing is performed by Travis.

Other changes:
- Build scripts were updated to take advantages of new Gradle and `config` features.
- The copyright year was updated to the current one.

The PR is going to be merged after the [PR](https://github.com/SpineEventEngine/base/pull/298) in `base`.

### CODE REVIEW

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: users/scripts/js.gradle [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/**
+ * The script declares tasks required to build a JavaScript module.
+ *
+ * <p>Extends and configures tasks from `config` submodule.
```
Comment (armiol) [inline users/scripts/js.gradle]: This file is also going to be repeated from project to project. I don't see why it's not a part of `config`.

File: version.gradle [hunk 2]
```diff
@@ -35,4 +35,6 @@ ext {
     spineBaseVersion = SPINE_VERSION
     spineVersion = SPINE_VERSION
     versionToPublish = SPINE_VERSION
+    // Have a different version for JS since NPM doesn't allow to publish the same version twice.
+    versionToPublishJs = '0.12.0'
```
Comment (armiol) [inline version.gradle]: Once `web` is published (see https://github.com/SpineEventEngine/web/pull/62) this should be `0.12.7`.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

**Pull request merged**